app/models/client_opportunity_match.rb

```diff
@@ -609,14 +609,6 @@ def reset_and_destroy!
     Matching::RunEngineJob.perform_later
   end
 
-  def top_priority?
-    return false if closed?
-    return false unless on_or_after_first_client_step?
-    return false unless opportunity.multiple_active_matches?
-
-    self == opportunity.prioritized_active_matches.first
-  end
-
```

Comment:
Let's leave this since it defines the logic for top_priority.  That way if we ever need to bring it back we won't have to dig too deeply.

Reply:
This has been added back in.